.github/actions/prepare-node-deps/action.yaml

```diff
@@ -52,3 +52,14 @@ runs:
         cp "$COMPASS_ICONS" "assets/fonts/"
         cp "$COMPASS_ICONS" "android/app/src/main/assets/fonts"
         echo "::endgroup::"
+
+    - name: ci/clone-APNG4Android
+      shell: bash
+      env:
+        APNG4Android: "node_modules/APNG4Android"
+      run: |
+        echo "::group::clone-APNG4Android"
+        if [ ! -d "$APNG4Android" ]; then
+          git clone -b "fix/MM-62375-oom-on-gif" --single-branch https://github.com/rahimrahman/APNG4Android.git "$APNG4Android";
```

Comment:
- [ ] TODO: Will replace this with the correct repo (mattermost/APNG4Android)

Reply:
Irrelevant in my opinion, but of it works here no need to change

Reply:
I like node_modules because it's easier to do the `npm run clean` and it'll auto-clean this library as well.  But I could have add an extra change to clean the folder wherever it might be in scripts/clean.sh as well.

I'll stick with node_modules.